maxim/logger/livekit/agent_session.py

```
@@ -160,7 +180,9 @@ def pre_hook(self, hook_name, args, kwargs):
         elif hook_name == "emit":
             if args[0] == "metrics_collected":
                 pass
-            scribe().debug(f"[Internal][{self.__class__.__name__}] emit called; args={args}, kwargs={kwargs}")
+            scribe().debug(
+                f"[Internal][{self.__class__.__name__}] emit called; args={args}, kwargs={kwargs}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)